library/blivet.py

```diff
@@ -608,8 +608,13 @@ def _create(self):
         if self._device:
             return
 
-        if safe_mode:
-            raise BlivetAnsibleError("cannot create new RAID in safe mode")
+        for spec in self._volume["disks"]:
+            disk = self._blivet.devicetree.resolve_device(spec)
+            if not disk.isleaf or disk.format.type is not None:
+                if safe_mode:
+                    raise BlivetAnsibleError("cannot create new RAID in safe mode")
```

Comment:
It would be better to use a slight variation on the error raised from `BlivetPool._create_members`.

Reply:
Fixed: Used slightly modified version of error from `BlivetPool._create_members`.